### PR TITLE
Switch back to htmx

### PR DESCRIPTION
**Pull request by thuiop**

### CODE REVIEW

File: templates/stats.ejs [hunk 1]
```diff
@@ -1,324 +1,297 @@
...
-                    document.cookie = stat + "=" + stats_aux[stat_aux];
-                }
-                // Refresh the page to reflect the changes
-                window.location.reload();
-            }
-        </script>
-      </head>
-      <body>
-        <main id="main">
-            <div class="header" style="justify-content: center;">
```
Comment (himanshunaidu) [inline templates/stats.ejs:146]: The missing header would mean that we would see the same header as for the main page.
I can fix that later, maybe by conditional headers.

Comment (himanshunaidu) [reply templates/stats.ejs:146]: or we could accept this as a new feature, where headers will be consistent throughout.
(But that would make the title header weird, as the Book and Chapter name may not be applicable for a specific page, so we can conditionally hide that depending on where we are.)

**Comment by himanshunaidu** [comment]
Header issue aside, I am fine with this PR.

**Review by himanshunaidu — APPROVED** [review]

File: templates/stats.ejs (hunk 1 above)
Comment (thuiop) [reply templates/stats.ejs:146]: Good catch; I added something that changes the header to "Stats" or "Menu" when you go on different pages.

**Pull request merged**